### ISSUE TITLE
Table: p-columnFilter missing aria-label

### ISSUE THREAD

**Issue by TaneliTuomola** [primefaces/primeng#14272]
### Describe the bug

WCAG 2.1: Button missing a text alternative. Provide a text alternative for the button p-columnFilter.

Add ariaLabel -property to the p-columnFilter component.

### Environment

Windows, Chrome

### Reproducer

https://primeng.org/table#filter

### Angular version

17.0.0

### PrimeNG version

17.0.0-rc.1

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.6.0
